plan.md

```diff
@@ -10,19 +10,51 @@ Note: to edit on HackMD https://hackmd.io/6fFy3_y6SOWonOYeGaNiHw?edit
 
 ## Content
 Why and when? Example applications.
+- Mention the iris example, pitch it as historical, but also as a
+  botanical and agriculture problem.
+  The benefit of this example is that it forces to think about
+  measurement, but also because it has one class that is easy to separate
+  only one feature
+    - Aurelie: real irises for the video?
+- The "adult" dataset
+    - Maybe looking at it with excel, to be in an environment familiar to
```

Comment:
Side-comment: my current feeling is that the audience we should target is roughly the one we have at Scipy tutorial. Even with this less ambitious target than we previously discussed, I think this is going to take a lot of more effort than we anticipate to have the material in a quality that is close to MOOC quality.

With that in mind, is excel still an environment familiar for our target audience, maybe, I don't know. It is true that some people coming to the Software Carpentry workshops are more familiar with excel and see the limitations of "manual" data analysis. I would still think that "Software Carpentry"-like audience is a too ambitious target for this MOOC.

Reply:
OK, so the scope is still open.

I don't think that it costs us a lot to have excel (or open office, because Excel would need an excel install, which I don't have).